src/utils/config.ts

```diff
@@ -40,6 +44,10 @@ export const SpidParams = t.intersection([
 
     ORG_NAME: NonEmptyString,
     ORG_URL: NonEmptyString,
+
+    JWT_TOKEN_EXPIRATION: NonNegativeInteger,
+    JWT_TOKEN_ISSUER: NonEmptyString,
+    JWT_TOKEN_PRIVATE_RSA_KEY: NonEmptyString,
```

Comment:
Are these properties required only if `ENABLE_JWT` is `true`?

Reply:
Yes, in case `ENABLE_JWT` is `false` we can fill our env variables with dummy values. What you you think about?

Reply:
You can use a custom type mapping this scenario and avoid to use dummy values:
```typescript
const JWTParams = t.union([
  t.interface({
    ENABLE_JWT: t.literal(true),
    JWT_TOKEN_EXPIRATION: NonNegativeInteger,
    JWT_TOKEN_ISSUER: NonEmptyString,
    JWT_TOKEN_PRIVATE_RSA_KEY: NonEmptyString,
  }), t.interface({
  ENABLE_JWT: t.literal(false),
})]);
type JWTParams = t.TypeOf<typeof JWTParams>;
```
and merge it into `IConfig`
```typescript
export const IConfig = t.intersection([
  t.interface({
    isProduction: t.boolean
  }),
  JWTParams,
  RedisParams,
  SpidParams
]);
```

Reply:
Thanks @BurnedMarshal :)

Reply:
Done in [54fbbec](https://github.com/pagopa/hub-spid-login-ms/pull/1/commits/54fbbecb2348fd01818728a356eb39516ca32125)